partcad-cli/poetry.toml

```diff
@@ -0,0 +1,3 @@
+[virtualenvs]
+create = true
+in-project = true
```

Comment:
This allows to store `.venv` on the host fs which makes starting of Dev Container faster.